Initial commit

ChenYiLins opened the pull request:
- This submission is related to: #925
- Components that are not used at present have been deleted. If necessary, please add them directly from the master branch to the current branch.

Spiritreader reviewed (dismissed):
You've configured your code formatter to expand short hand get setters

```c#
public AdmConfig Config { get; private set; }
```

to

```c#
public AdmConfig Config
{
    get; private set;
}
```

~~Could you revert this change?~~ I've adjusted the formatter settings.
I don't really mind if this is done in the AutoDarkModeApp, but for the other components I would prefer it to remain unchanged.

Spiritreader commented:
> Components that are not used at present have been deleted. If necessary, please add them directly from the master branch to the current branch.

Also, merging this initial commit should only be done once all non UI components have been re-added in their previous (or future agreed upon) state

ChenYiLins commented:
> > Components that are not used at present have been deleted. If necessary, please add them directly from the master branch to the current branch.
> 
> Also, merging this initial commit should only be done once all non UI components have been re-added in their previous (or future agreed upon) state

You are right, this can avoid many repeated commits.

> I've gone ahead and re-added the missing backend files, 和 separated the winui3 build from the backend for now, such that it's easier to work on it.
> 
> Also the target has been upped to .net9
> 
> In addition, I've adjusted the code formatter rules to match the previous style for single line auto properties and added a few more for namespace formatting.
> 
> Please check if everything still compiles, you might need to manually delete the `obj` directories within the modules and the `Debug` directory.
> 
> So in that regard, the initial PR is ready to merge into the winui3 branch. I will be waiting for further comments.

Thank you for adding.

It is wise to upgrade to .net9, because it can promote the realization of Aot. At the same time, need to make changes to the code of ViewModels.

A unified code style can make the code easier to maintain. If necessary, additional documents can be added to explain.

Jay-o-Way commented:
Cross-referencing https://github.com/ChenYiLins/Windows-Auto-Night-Mode/pull/1

Jay-o-Way commented:
@Spiritreader @Armin2208 at what point would you like to pull this in? Then I can use the adm repo to base a next pr on.

ChenYiLins commented:
> Also, merging this initial commit should only be done once all non UI components have been re-added in their previous (or future agreed upon) state

@Jay-o-Way This counld avoid many seemingly repetitive commit before the basic work is completed.

ChenYiLins commented:
@Jay-o-Way Do you run in package? Please make sure you are running in unpackage. Maybe I should delete it.

Spiritreader commented:
> Now I get this :( ![image](https://private-user-images.githubusercontent.com/65828559/428939511-a1a3614c-e34f-4356-9eb3-d5560d229091.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDM0OTg4NTUsIm5iZiI6MTc0MzQ5ODU1NSwicGF0aCI6Ii82NTgyODU1OS80Mjg5Mzk1MTEtYTFhMzYxNGMtZTM0Zi00MzU2LTllYjMtZDU1NjBkMjI5MDkxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDAxVDA5MDkxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRjYWY0NTgxYjI1ODZjZmM5MjFkZTFjMjg0ODVjM2E0MzUyNTBlZGYzMjA3YWQyNDY5NmUxNDNkMmU0YmFkYjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.OKCrjkLW4bOvIa7eTZbmSkquk8b_iJR2_A9pPB7T5zg)

Yes, it seems like you've been running the packaged (msix) version so far.
Make sure the build is on unpackaged. Also selecting yes on that message should not break anything.
